File: 0705-design-hashset/0705-design-hashset.js
var MyHashSet = function() {
  
};

MyHashSet.prototype.add = function(key) {
    this[key] = null;
};

MyHashSet.prototype.remove = function(key) {
    delete this[key]
};

MyHashSet.prototype.contains = function(key) {

    return this.hasOwnProperty(key)
};

File: 0705-design-hashset/0705-design-hashset.py
class MyHashSet:

    def __init__(self):
        self.d = {}

    def add(self, key: int) -> None:
        self.d[key] = 1

    def remove(self, key: int) -> None:
        self.d[key] = 0

    def contains(self, key: int) -> bool:
        return self.d.get(key,0)!=0